requirements.txt

```diff
@@ -1,6 +1,6 @@
 pytorch_lightning
 jsonargparse[signatures]
-git+https://github.com/Lightning-AI/lightning.git@training_studio_fixes  # TODO: replace with lightning once 0.6.0 is released
+git+https://github.com/Lightning-AI/lightning.git@reduce_cost  # TODO: replace with lightning once 0.6.0 is released
```

Comment:
0.6.0 released?
https://github.com/Lightning-AI/lightning/releases/tag/app%2F0.6.0

Reply:
Yes, this is out-dated. Remove the fixes.